test/crypto/index.js

```diff
@@ -12,16 +12,14 @@
  * Removal or modification of this copyright notice is prohibited.
  *
  */
+import cryptoModule from '../../src/crypto/index';
```

Comment:
Doesn't it look weird? Please, have a look on this move.

Reply:
I guess because of the license headers GitHub thought it is a file movement, even though one was created and one was deleted.